x-pack/legacy/plugins/siem/public/components/embeddables/actions/apply_siem_filter_action.tsx

```diff
@@ -50,37 +41,6 @@ export class ApplySiemFilterAction implements IAction<ActionContext> {
     if (!filters) {
       throw new TypeError('Applying a filter requires a filter as context');
     }
-
-    // Parse queryExpression from queryDSL and apply to SIEM global KQL Bar via redux
-    const filterObject = getOr(null, '[0].query.match', filters);
-
-    if (filterObject != null) {
-      const filterQuery = getOr('', 'query.query', embeddable.getInput());
-      const filterKey = Object.keys(filterObject)[0];
-
-      const filterExpression = getFilterExpression(filterKey, filterObject[filterKey].query);
-
-      this.applyFilterQueryFromKueryExpression(
-        filterQuery.length > 0 ? `${filterQuery} and ${filterExpression}` : filterExpression
-      );
-    }
+    siemFilterManager.addFilters(filters);
```

Comment:
This new implementation makes filtering from the map so much cleaner -- looks good @XavierM! Now that we have the new filter bar I'll have to look into re-enabling the 'draw to filter' feature that maps provide.